### DISCUSSION TITLE
📝【LeetCode】在实际问题中运用二分查找模板代码

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#81] [Announcements]
https://imageslr.github.io/2020/03/15/leetcode-875.html 

 引言 在上一篇文章中，我们介绍了「二分查找」的通用模板，并通过几个例题说明了如何套用这个模板。建议读者先阅读上一篇文章：一个模板通杀所有「二分查找」问题。

**Comment by godlun** [comment]
感谢！

**Comment by Alphasxd** [comment]
博主你好，这个博客关于code的渲染的代码缩进中是区分‘tab’和‘4个空格’的，因为我也套用了这个jekyll主题，所以发现了这个点
